src/test/java/com/teragrep/pth10/steps/teragrep/bloomfilter/ToBloomFilterTest.java

```diff
@@ -131,7 +132,7 @@ void testEquality() {
     @Test
     void testEqualityCacheFilled() {
         ToBloomFilter filter = new ToBloomFilter(bytes);
-        Assertions.assertTrue(filter.mightContain("one"));
+        filter.fromBytes();
```

Comment:
what happened to mightContain assertion?

Reply:
Somehow got deleted when refactoring toBloomFilter.